### PR TITLE
Multiple Pythons content

### PR DESCRIPTION
**Pull request by hroncok**
<del>Note that this is not yet true, so the ready flag is still false.</del>

### CODE REVIEW

**Comment by hroncok** [comment]
<del>Also, the link leads to 404, but that will be OK once https://github.com/developer-portal/content/pull/159 is merged.</del>

**Comment by hroncok** [comment]
Ready to review and merge.

File: points.yml [hunk 1]
```diff
@@ -9,15 +9,18 @@
   link:
     href: http://example.com
     caption: Find out more!
-- name: All the Pythons
-  ready: false
+- name: Multiple Pythons
```
Comment (torsava) [inline points.yml]: Just a small note: I feel the original name was better, more enthusiastic. "Multiple Pythons" sounds a little bit dry.

File: points.yml [hunk 2]
```diff
@@ -9,15 +9,18 @@
...
   content: |
-    We have many varieties of Python in Fedora!
-    Pick one, or test them all!
+    Fedora has CPyhtons in multiple 3.X versions, 2.7 and 2.6, PyPys, Jython
+    and even MicroPython, all available in the repositories.
```
Comment (torsava) [inline points.yml]: The word CPython should not be used in the plural form. How about:

> Fedora includes a great many versions of Python: CPython in multpile 3.X and 2.X versions, PyPy 2 and 3, Jython and even MicroPython, all available in the repositories.

File: points.yml [hunk 3]
```diff
@@ -9,15 +9,18 @@
...
+    Testing Python code on different interpreters has never been easier.
+
+    All you have to do is `sudo dnf install tox`, or `dnf install` just the
+    ones you need.
```
Comment (torsava) [inline points.yml]: I'd change the order for more clarity:

> You can `sudo dnf install` the Python you need, or do `sudo dnf install tox` to get them all.

**Review by torsava — COMMENTED** [review]
Some suggestions.

**Comment by hroncok** [comment]
Added a fixup commit.

**Comment by torsava** [comment]
LGTM.

**Pull request merged**